src/vivarium_gates_nutrition_optimization/data/loader.py

```diff
@@ -145,7 +150,43 @@ def _load_em_from_meid(location, meid, measure):
     return vi_utils.sort_hierarchical_data(data).droplevel("location")
 
 
-# TODO - add project-specific data functions here
+##################
+# Pregnancy Data #
+##################
+
+
+def get_pregnancy_end_rate(location: str) -> pd.DataFrame:
+    asfr = get_data(data_keys.PREGNANCY.ASFR, location)
+    sbr = get_data(data_keys.PREGNANCY.SBR, location)
+    sbr = sbr.reset_index(level="year_end", drop=True).reindex(
+        asfr.index, level="year_start", fill_value=0.0
+    )
+    incidence_c995 = get_data(data_keys.PREGNANCY.INCIDENCE_RATE_MISCARRIAGE, location)
+    incidence_c374 = get_data(data_keys.PREGNANCY.INCIDENCE_RATE_ECTOPIC, location)
+    pregnancy_end_rate = (
+        asfr + asfr.multiply(sbr["value"], axis=0) + incidence_c995 + incidence_c374
+    )
+    return pregnancy_end_rate.reorder_levels(asfr.index.names)
+
+
+def load_asfr(key: str, location: str) -> pd.DataFrame:
+    asfr = load_standard_data(key, location)
+    asfr = asfr.reset_index()
+    asfr_pivot = asfr.pivot(
+        index=[col for col in metadata.ARTIFACT_INDEX_COLUMNS if col != "location"],
+        columns="parameter",
+        values="value",
+    )
+    seed = f"{key}_{location}"
+    # Keep lognormal draws from Gates IV Iron until @alibow says otherwise
```

Comment:
nit: make this a `#FIXME`

Reply:
I'll just delete the comment, since now it sounds like lognormal is correct.